### ISSUE TITLE
Compilation failure on aarch64

### ISSUE THREAD

**Issue by ctron** [fede1024/rust-rdkafka#445]
The compilation fails with:

```
error[E0308]: mismatched types
   --> /usr/src/.cargo-container-home/registry/src/github.com-1ecc6299db9ec823/rdkafka-0.25.0/src/config.rs:153:17
    |
153 |                 buf.as_mut_ptr() as *mut i8,
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*mut u8`
               found raw pointer `*mut i8`
```

**Comment by jszwedko** [comment]
I do see this error in 0.25.0 but 0.28.0 seems fine. It looks like the fix was in 0.26.0: https://github.com/fede1024/rust-rdkafka/blob/master/changelog.md#0260-2021-03-16

**Comment by ctron** [comment]
Indeed. Upgrading did solve the issue.
